CMakeLists.txt

```diff
@@ -18,15 +22,6 @@ set(CMAKE_CXX_STANDARD 11)
 # iGeom assumed to have cone functionality
 add_definitions(-DHAVE_IGEOM_CONE=ON)
 
-# Adjust compiler setting for Linux using gcc version 5.0 and higher
-if (CMAKE_SYSTEM_NAME MATCHES Linux AND CMAKE_COMPILER_IS_GNUCXX)
-  if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "5.0")
-    if (NOT CMAKE_CXX_FLAGS MATCHES _GLIBCXX_USE_CXX11_ABI)
-      set(CMAKE_CXX_FLAGS "-D_GLIBCXX_USE_CXX11_ABI=0 ${CMAKE_CXX_FLAGS}")
```

Comment:
Perhaps I am out of the loop, but how come these lines were removed? I remember them being necessary when I was building on Linux.

Reply:
it is still needed for Trelis 17.1.
And we now add it manually on the reps we build (MOAB, and DAGMC).

but as this need to be removed for Cubit 2020.2, we decided that we remove them and one specify them manually in the make process.
(@pshriwise I think we need to update the doc)